src/components/Amalgamation/AmalgamationStatement.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-card
+    id="amalgamation-statement"
+    flat
+  >
+    <v-row>
+      <v-col
+        cols="12"
+        sm="3"
+        class="pr-4"
+      >
+        <label>
+          Amalgamation Statement
+        </label>
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="9"
+      >
+        <v-radio-group
+          v-model="courtApproval"
+          class="mt-0 pt-0"
+          @change="setCourtApproval()"
+        >
+          <v-radio
+            class="radio-button"
+            label="With Court Approval"
+            :value="true"
+          />
+          <div
+            class="ml-8 statement-text"
+          >
+            This amalgamation has been approved by the court and a copy of the
+            entered court order approving the amalgamation has been obtained
+            and has been deposited in the records office of each of the
+            amalgamating companies.
+          </div>
+
+          <v-radio
+            class="pt-4 radio-button"
+            label="Without Court Approval"
+            :value="false"
+          />
+          <div
+            class="ml-8 statement-text"
+          >
+            This amalgamation has been effected without court approval.
+            A copy of all of the required affidavits under section 277(1)
+            have been obtained and the affidavit obtained from each
+            amalgamating company has been deposited in that company’s
+            records office.
+          </div>
+        </v-radio-group>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Emit } from 'vue-property-decorator'
+import { AmalgamationStateIF } from '@/interfaces/store-interfaces/state-interfaces/amalgamation-state-interface'
+import { Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+
+@Component({})
+export default class AmalgamationStatement extends Vue {
+  @Getter(useStore) getAmalgamationCourtApproval!: boolean
+
+  // Local properties
+  courtApproval: AmalgamationStateIF['courtApproval'] = null
+
+  /** Called when component is mounted. */
+  mounted (): void {
+    if (this.getAmalgamationCourtApproval != null) {
+      this.courtApproval = this.getAmalgamationCourtApproval
+    }
```

Comment:
1. Why do you need the conditional on line 76? If the getter value is null then you could simply reassign null to the local variable.

2. Always use `===` or `!==` in comparisons.

3. This fails if the component is mounted before the getter has a valid value (ie, the draft filing has been loaded). Fortunately, I think App.vue is written so that the pages aren't displayed (mounted) until the draft filing has been loaded, so it works here, but don't count on it in other places. I think we don't have a consistent method of having local variables synced with props or store variables (which are both read-only), so this is always a bit tricky. In other code, we have a watcher that sets the local initially ("immediate: true") and whenever the prop/getter changes. I think that's safer code but I think it's still overly complex. If you think of a nicer way to do this, let me know.

Reply:
> 1. Why do you need the conditional on line 76? If the getter value is null then you could simply reassign null to the local variable.

yeahhh, I just realize that.

> 3.

I think I will keep the same with other file, using ("immediate: true"). But I am wondering what does "other places" in ` I think App.vue is written so that the pages aren't displayed (mounted) until the draft filing has been loaded, so it works here, but don't count on it in other places. ` means?

Reply:
> But I am wondering what does "other places" in (snip) means?

In this web app, I think it is safe to assume that most components aren't created/mounted until the data is loaded, but in a different UI this may not be true. Maybe Filings UI, maybe a future web app you work on, etc :)

Reply:
Looks good.